README.md

```
@@ -1 +1,179 @@
+
+# Clean up unnecessary packages
+echo "Cleaning up unnecessary packages..."
+sudo apt autoremove -y
+
+echo "Java and Maven installation completed successfully!"
+
+mvn clean install : it will build the app
+
+mvn spring-boot:run : this would run the jar file
+=============================================================================================================================================================================
+http://ec2-ip:8080/petclinic/pets
```

Markdownlint (0.37.0)

Expected: atx; Actual: setext
Heading style

(MD003, heading-style)